ambry-api/src/main/java/com/github/ambry/account/ContainerBuilder.java

```diff
@@ -248,8 +248,8 @@ public ContainerBuilder setReplicationPolicy(String replicationPolicy) {
   }
 
   /**
-   * Sets the whitelist for the content types for which filenames can be sent on download
-   * @param contentTypeWhitelistForFilenamesOnDownload the whitelist for the content types for which filenames can be
+   * Sets the allow list for the content types for which filenames can be sent on download
+   * @param contentTypeWhitelistForFilenamesOnDownload the allow list for the content types for which filenames can be
```

Comment:
should we also consider replace the variable name?

Reply:
My bad, revert the changes for the comments. will modify the name for the future pr.